Userland/Libraries/LibCore/CMakeLists.txt

```diff
@@ -57,6 +56,13 @@ set(SOURCES
     Timer.cpp
     UDPServer.cpp
 )
+
+if (WIN32)
+    list(APPEND SOURCES CommandWindows.cpp)
```

Comment:
the only use for Core::Command in the codebase at the moment is in the test262-runner file. We've been moving towards using Core::Process as much as possible instead. So, I would rather remove Core::Command, and centralize the process management code you've written here into Core::Process instead.